rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/shared/SerializationType.java

```diff
@@ -25,16 +25,22 @@
  */
 package org.ow2.proactive_grid_cloud_portal.rm.shared;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)